addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/studies/StudyPermissionsTable.js

```diff
@@ -103,7 +107,8 @@ class StudyPermissionsTable extends React.Component {
 
   renderTable() {
     const studyPermissions = this.permissionsStore.studyPermissions;
-    const isEditable = studyPermissions.adminUsers.some(uid => uid === this.currUser.uid);
+    const isEditable =
+      studyPermissions.adminUsers.some(uid => uid === this.currUser.uid) && this.study.state.canChangePermission;
```

Comment:
Is it possible that you move this existing logic as a view on the permissions entity model?

something like  isAdmin(uid)  as a view on the permissions entity model?

Reply:
Emmm, view methods can not have pass in parameter

Reply:
Ah, my bad. I thought view method can only be `get` something, which can't have pass in parameter. Will update it.